actions/changed-files/check_changed_files.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+# THIS expects two environment variables to be present
+#  - INCLUDE_PATTERN
+#  - EXCLUDE_PATTERN
```

Comment:
I just realized that these are called `*_PATTERN`, but they actually contain multiple patterns, so we might want to rename this with the trailing `S` to avoid confusion

Reply:
I am using pattern with 's` while splitting it . That is why i removed s. we can keep it like this if it is not a concern